dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilder.java

```diff
@@ -183,15 +183,13 @@ private DependencyNode resolveCompileTimeDependencies(
     for (RemoteRepository repository : repositories) {
       collectRequest.addRepository(repository);
     }
-    CollectResult collectResult = system.collectDependencies(session, collectRequest);
```

Comment:
This was unnecessary, because resolveDependencies (line 191) `is essentially a combination of {@link #collectDependencies(RepositorySystemSession, CollectRequest)} and {@link #resolveArtifacts(RepositorySystemSession, Collection)}.` as per the [Javadoc](https://maven.apache.org/resolver/maven-resolver-api/apidocs/org/eclipse/aether/RepositorySystem.html#resolveDependencies(org.eclipse.aether.RepositorySystemSession,%20org.eclipse.aether.resolution.DependencyRequest)).